### ISSUE TITLE
Using the Telis 4 RTS remote with multiple blinds ?

### ISSUE THREAD

**Issue by fabienpatou** [beele/HomebridgeSomfyRemote#8]
Hi,

Opening an issue as I don't find this documented.

Is it possible with this plugin to control multiple blinds paired with the same Telis 4 RTS remote ?

Or do I need to wire all Telis 1 RTS remotes to the RPI ?

Thanks,
Fabien

**Comment by stefanluyten** [comment]
I wonder the same.... did you find solution already?

Thanks
Stefan

**Comment by brancooo1** [comment]
No, this plugin [allegedly doesn't support that](https://github.com/beele/HomebridgeSomfyRemote/issues/6). 
But I'll be preparing plugin for new 5 channel Somfy remotes.

**Comment by Peet-B** [comment]
> 

Is there any update on this feature? I have a spare Telis 4 remote and 4 blinds around the house. It would be awesome to control them separately without having to dish out €70,- for their own hub if I could hack something together myself...
